tests/e2e/utils/src/page-utils.js

```diff
@@ -209,6 +209,19 @@ const selectOptionInSelect2 = async ( value, selector = 'input.select2-search__f
 	await page.keyboard.press('Enter');
 };
 
+/**
+ * Search by any term for an order
+ *
+ * @param {string} value Value to be entered into the search field
+ * @param {string} orderId Order ID
+ */
+const searchForOrder = async (value, orderId) => {
+	await clearAndFillInput('#post-search-input', value);
+	await expect(page).toClick('#search-submit');
+	await page.waitForSelector('#the-list');
+	await expect(page).toMatchElement('.order_number > a.order-view', {text: '#'+orderId+' John Doe'});
```

Comment:
It might be worth making the name here a variable that can be passed in as well, so something like:

```suggestion
	await expect(page).toMatchElement('.order_number > a.order-view', {text: '#${orderId} ${customerName}'});
```

Reply:
I agree, will change that 👍